docker-compose.systemtest.yml

```diff
@@ -4,7 +4,6 @@ services:
   postgresql:
     image: odinsmr/odin_db:latest
     ports: ["5432"]
-    restart: unless-stopped
```

Comment:
why are these removed? Isn't that super annoying if I run the system-tests locally?

Reply:
If a container stops while testing there's probably something wrong.

Reply:
:facepalm: I was thinking `restart: always` was the default. I was confused.